server/routes/todo.js

```diff
@@ -10,6 +10,9 @@ router.post("/todo", todoController.todo_post);
 router.get("/todo", todoController.todo_get);
 
 // Create a route to mark item as complete (we could also use this if we wanted to mark the item as not complete)
-router.put("/todo/:id", todoController.todo_complete);
+router.put("/todo/update/:id", todoController.todo_complete);
+
+// Delete todo item
+router.post("/todo/delete/:id", todoController.todo_delete);
```

Comment:
This should be a router.delete() call instead. And similar to my other comment, the path can just be a description of the resource you're looking for so it should be "/todo/:id".

Reply:
I was also wondering about this. Thanks man. Will update